app/time_tracking/vacation/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Vacation(models.Model):
+    brief_description = models.CharField(max_length=120, blank=True, null=True)
+    start_date        = models.DateField()
+    number_of_days    = models.PositiveSmallIntegerField()
```

Comment:
`number_of_days` could be how the user specify the vacation, but from modeling perspective it must be an end date

Reply:
Sounds good.
Due to the fact that SQLite stores the date/time as strings, I thought of storing the `start_date` and `end_date` as UnixTime for faster queries operations, and converting the date to UnixTime on the serializer so that the user would send and receive a human representative date,
but the max number of records that I'll have to iterate throw on the server-side is only 16, therefore, I'll store the start and end dates as `DateField` for now.